bayes_opt/__init__.py

```diff
@@ -1,13 +1,16 @@
 """Pure Python implementation of bayesian global optimization with gaussian processes."""
-from .bayesian_optimization import BayesianOptimization, Events
-from .domain_reduction import SequentialDomainReductionTransformer 
-from .logger import ScreenLogger, JSONLogger
-from .constraint import ConstraintModel
-from . import acquisition
+
+from __future__ import annotations
```

Comment:
I've had a look and it seems that this import is only needed when annotating with some not-yet-defined type. Is this necessary here?

Reply:
Not necessary.
Automatically generated based on the `ruff` rule. 
You can change the `ruff` rule if you wish, or exclude it from the file.